_faq/apply_for_two_orgs.md

```diff
@@ -0,0 +1,6 @@
+---
+question: "Applicants should try to apply for two orgs!"
+---
+Most applicants to coala will be unsuccessful, due to high demand. So, you need to be very well prepared, as there will be 60+ applications for 10 positions, and 30 of those applications will be of very high quality. 
+
+For best results, applicants should also apply at a second organization.
```

Comment:
edit this to: `As a precaution, you can apply in other organisations as well.`

and the question must be in "question" form
like: `Should I apply in two organisations?`